app/Helpers/PushNotification.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
remove the following commented code once the application is working on mobile app